product_sale_manufactured_for/models/res_partner.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from collections import defaultdict
+
+from odoo import _, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def write(self, vals):
+        res = super().write(vals)
+        if vals.get("active") is False:
+            query = (
```

Comment:
A nice comment here would be nice. Even better: an additional method with its own docstring :wink:

Reply:
Added